AK/BuiltinWrappers.h

```diff
@@ -97,6 +97,24 @@ inline constexpr int count_leading_zeroes(IntType value)
 #endif
 }
 
+#ifdef __SIZEOF_INT128__
+// This is required for math.cpp internal_scalbn
+inline constexpr int count_leading_zeroes(unsigned __int128 value)
+{
+#    if defined(AK_COMPILER_CLANG) || defined(AK_COMPILER_GCC)
```

Comment:
Do we support any other compilers? Maybe enough to just do this branch unconditionally for now?

Reply:
It's like this everywhere else. And I don't know if this works for other compilers.